### ISSUE TITLE
Missing dependencies

### ISSUE THREAD

**Issue by mleece** [greyhavens/bang-game#1]
To whom it may concern,

I decided to try building this on my own to tinker around with it (hope that's ok!), but am running into a dependency error when I try to mvn compile. The error that I get is (reformatted hopefully for readability):
```
[ERROR] Failed to execute goal on project bang: Could not resolve dependencies for project com.threerings:bang:jar:1.1-SNAPSHOT: The following artifacts could not be resolved: com.jme:jme:jar:1.1, 
com.jme:jme-awt:jar:1.1, 
com.jme:jme-effects:jar:1.1, 
com.jme:jme-gamestates:jar:1.1, 
com.jme:jme-model:jar:1.1, 
com.jme:jme-scene:jar:1.1, 
com.jme:jme-sound:jar:1.1, 
com.jme:jme-terrain:jar:1.1, 
com.samskivert:jme-bui:jar:1.1-SNAPSHOT, 
com.threerings:threerings:jar:1.0, 
com.threerings:hemiptera:jar:1.0: 
Failure to find com.jme:jme:jar:1.1 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
```

Since then, I've found some copies of the jme jars that at least make the dependencies go away if I manually add them to my .m2 folder, but I don't have a solution for the samskivert jar or the threerings jar (and I'm not confident that the jme jars will actually work beyond simply satisfying the dependency).

This is using a fresh install of Maven, version info (in case it's relevant):
Apache Maven 3.3.9
Maven home: /usr/share/maven
Java version: 1.8.0_03-Ubuntu, vendor: Oracle Corporation
Java home: /usr/lib/jvm/java-8-openjdk-amd64/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "linux", version: "4.4.0-21-generic", arch: "amd64", family: "unix"

Hopefully someone is still watching this page! If so, thanks for the dedication!

EDIT: I've also found the threerings:threerings.jar:1.1 that was posted in the msoy issues, and am assuming that that's backwards compatible. The only thing that I cannot find is hemiptera.

**Comment by samskivert** [comment]
I just pushed a big build system revamp that I was working on a year ago or so, and now everything should build (using Gradle). I even added basic instructions to the top-level README.

I just tried everything out on a machine with no secret ancient depends lying around, so it should work, but I can't promise that the road won't be bumpy!

**Comment by mleece** [comment]
Since there's no messaging on Github, just wanted to comment here saying thanks for everything! Got my local server up and running and can login successfully. 

The only small bumps that I ran into: 
- tools/build.gradle calls for nenya-tools:1.7, when it only goes up to 1.6.1 it seems. I switched it with the line above it and was fine.
- For some reason I needed to change the paths in assets/build.gradle on line 90 from relative paths to absolute paths. That one I can't explain, but it worked after I made the switch.

Again, thanks for the help on this old code. I know it's not necessary, and it's appreciated all the more for that fact.
